class.jetpack-debugger.php

```diff
@@ -204,8 +206,9 @@ public static function jetpack_debug_display_handler() {
 					</div>
 					<hr />
 					<div id="sync-related-posts">
-						<p><?php echo esc_html__( 'Some features of Jetpack use the WordPress.com infrastructure and require that your public content be mirrored there. If you see intermittent issues only affecting certain posts, please try requesting a reindex of your posts.', 'jetpack' ); ?></p>
-						<?php echo Jetpack::init()->sync->reindex_ui() ?>
+						<p><?php echo esc_html__( 'Some features of Jetpack uses the WordPress.com infrastructure and requires that your public content be mirrored there. If you see intermittent issues only affecting certain posts, please try requesting a reindex of your posts.', 'jetpack' ); ?></p>
```

Comment:
This is overwriting a change that was made in b4bc0ed4b768869433e1994da11a59b773701301